setup.py

```diff
@@ -145,4 +145,5 @@ def get_long_description():
                   ],
               },
           zip_safe=False,
-          test_suite='selftests')
+          test_suite='selftests',
+          python_requires='>=2.6')
```

Comment:
Yep, it simply logs warning,

```
/usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)

```

and newer version should support it correctly...